app/models/project_media.rb

```diff
@@ -295,14 +302,42 @@ def replace_by(new_pm, skip_send_report = false)
       Rails.cache.write("check_cached_field:ProjectMedia:#{new_pm.id}:creator_name", 'Import')
 
       # Apply other stuff in background
-      self.class.delay_for(5.seconds).apply_replace_by(self.id, new_pm.id, skip_send_report)
+      options = {
+        author_id: User.current&.id,
+        assignments_ids: assignments_ids,
+        skip_send_report: skip_send_report
+      }
+      self.class.delay_for(5.seconds).apply_replace_by(self.id, new_pm.id, options.to_json)
```

Comment:
Suggestion: Since you're here, I think this interval can be lower, right?

Reply:
right